TTSegmentedControl/Models/TTSegmentedControlTitle.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  TTSegmentedControlTitle.swift
+//  TTSegmentedControl
+//
+//  Created by Igor Dumitru on 02.02.2023.
+//
+
+import UIKit
+
+public struct TTSegmentedControlTitle {
+    public let text: String?
```

Comment:
aici s-ar putea sa creeze confuzie faptul ca avem si `text` cu color si font, si i se face override daca setam attributedText. @EfraimB @dumitruigor ce ziceti?

Reply:
Da, am putea folosi un enum si aici

Reply:
Ideea e ca daca tot schimbam API-ul hai sa il facem right.

Reply:
@CataD256 @EfraimB 
`     let label = UILabel()
        label.text = "text"
        label.textColor = .red
        label.font = UIFont.systemFont(ofSize: 20)
        label.attributedText = NSAttributedString(string: "text")`

Reply:
ai dreptate, dar la UILabel, attributedText inlocuieste si text. E logica un pic diferita si nu e pe initializer.

`label.attributedText = NSAttributedString(string: "updated text"); label.text = "text"`
In exemplul asta textul vizibil e "text"
eu as face fie 2 initializere diferite, fie as pune un enum cu 2 case-uri, ca sa nu fie si attributedText si text in acelasi call.